code/src/components/TodoList.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react'
+import { useSelector, useDispatch } from 'react-redux'
+
+import todos from '../reducers/todos'
+
+import RemoveTodo from './RemoveTodo'
+import TodoItem from './TodoItem'
+
+const TodoList = () => {
+  const items = useSelector((store) => store.todos.items)
+  const dispatch = useDispatch()
+
+  return (
+    <div className="todo-list-wrapper">
+      {items.map(todo => (
+        <div className="todo-list-container" key={todo.id}>
```

Comment:
I think "todo-list-container" is a bit misleading, because it's actually more of a "todo-item-container", as you're creating one of these for each todo.